railties/lib/rails/generators/rails/encrypted_secrets/templates/config/secrets.yml.enc

```diff
@@ -0,0 +1,2 @@
+production:
+  secret_key_base: <%= SecureRandom.hex(64) %>
```

Comment:
This seems problematic for containers. This value will always be different for every app instance that boots, meaning that sessions between different servers will not be able to submit forms to each other. 

Right now we default to generating secrets.yml with a `ENV['SECRET_KEY_BASE']` and it works out of the box as long as you have that env var set. Could we maybe make it so that this file is optional? I.e. if you've only got a `RAILS_MASTER_KEY` set, use that as the secret key base?